recipes/kit4dl/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "kit4dl" %}
+{% set version = "2023.08b0" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  path: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/kit4dl-{{ version }}.tar.gz
+  sha256: 7ab8579afdf042b32559b9cc8eac72ed34d3dbd1fba87e0dfa09353cf7d65ce4
+
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  build:
+    - python >=3.10
+  host:
+    - python >=3.10
+    - pip    
+  run:
+    - "python >=3.10"
+    - "pytorch"
+    - "lightning"
+    - "torchmetrics"
+    - "numpy >=1.23.0"
+    - "pydantic >=1.10.0"
+    - "scikit-learn >=1.2.0"
+    - "colormath"
+    - "tqdm"
+    - "typer[all]"
```

Comment:
conda doesn't support package "extras" like `[all]`. You would need to add the requirements here. See:
https://github.com/tiangolo/typer/blob/master/pyproject.toml#L67-L71

Note: These are the requirements from the latest revision on Github, so you should check to see if the version you're attempting to build from is the same.